CONTRIBUTING.md

```diff
@@ -29,6 +29,50 @@ submitted for inclusion in the work by you, as defined in the
 Apache-2.0 license, shall be dual licensed as in the README, without any
 additional terms or conditions.
 
+## Version numbers
+
+We use `git-testament` to construct our version strings. This includes into
+the binary details of the git commit, tag description, and also an indication
+of modifications to the working tree present when the binary was compiled.
+
+This means that in normal development, you might get information from invoking
+`rustup --version` which look like `rustup-init 1.18.3+15 (a54051502 2019-05-26)`
```

Comment:
```suggestion
`rustup --version` which looks like `rustup-init 1.18.3+15 (a54051502 2019-05-26)`
```

Reply:
I suggest dropping the 'This means'. Just say 'In normal development you may ...', as you have changes to make anyway.

Reply:
Agreed, done.